src/help-widget.spec.ts

```diff
@@ -355,4 +355,30 @@ describe('BBHelpHelpWidget', () => {
     expect(window.console.error).toHaveBeenCalledWith(`No matching response for action: ${testAction}`);
     done();
   });
+
+  it('should assess what\'s new revision and return revision number', () => {
+    const fakeConfig = {
+      productId: 'rex',
+      whatsNewRevisions: 'rex=30;'
+    };
+    helpWidget.load(fakeConfig);
+    expect(helpWidget.getWhatsNewRevision()).toEqual(30);
+  });
+
+  it('should assess what\'s new revision with no found revision and return 0', () => {
```

Comment:
I would also add one more test to make sure it's finding the correct revision amongst a string of revisions.  The method splits a string by `;` and then searches for a match on productId.  Something like 
```
  it('should assess what\'s new revision and return revision number from several revisions', () => {
    const fakeConfig = {
      productId: 'fe',
      whatsNewRevisions: 'rex=30;fe=10;lo=15'
    };
    helpWidget.load(fakeConfig);
    expect(helpWidget.getWhatsNewRevision()).toEqual(10);
  });
```